app-modules/service-management/src/Notifications/Concerns/HandlesServiceRequestTemplateContent.php

```diff
@@ -131,4 +134,30 @@ protected function injectButtonUrlIntoTiptapContent(array $content, ?ServiceRequ
 
         return $content;
     }
+
+    protected function getRecentUpdateFormatted(): string
+    {
+        $recentUpdate = $this->serviceRequest
+            ->serviceRequestUpdates()
+            ->orderBy('updated_at', 'desc')
+            ->first();
+
+        if (! $recentUpdate) {
+            return 'No updates available';
+        }
+
+        if ($recentUpdate->createdBy instanceof Contact) {
+            $contact = $recentUpdate->createdBy;
+            $organizationName = $contact->organization ? $contact->organization->name : 'N/A';
+            $creatorInfo = "{$contact->full_name} - {$organizationName}";
+        } else {
+            $user = $recentUpdate->createdBy;
+            assert($user instanceof User);
+            $creatorInfo = "{$user->name} - Service Provider";
+        }
```

Comment:
One thing to note is that Product did make this ticket dependent on https://canyongbs.atlassian.net/browse/AIDAPP-705 because they wanted the timezone application done.

They will want the timezone applied from `app/Settings/DisplaySettings.php` `getTimezone`.